### ISSUE TITLE
R-CMD check test error

### ISSUE THREAD

**Issue by sebdalgarno** [LandSciTech/ccviR#177]
openjournals/joss-reviews#7150 

I'm getting some test errors in `test-run_ccvi_app.R`

Can you check on your end?

**Comment by see24** [comment]
Yes I get an error on my end as well. I have had a lot of trouble with shinytest and have since given up on it. I would like to leave the file in case I am able to get it working again. For now I just have some manual tests that I run for the app (See #151) . All the non-shiny package functions have testthat tests though.

**Comment by sebdalgarno** [comment]
would it be possible to remove (or comment out) the stop call in this test so that all tests pass?
```
test_that("save works especially for spatial and range change and all related inputs", {
  skip_on_cran()
  skip_on_ci()
  stop("add test")
})
```
